composer.json

```diff
@@ -59,7 +59,7 @@
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
     "suggest": {
-        "liip/imagine-bundle": "^0.9",
+        "liip/imagine-bundle": "^1.7",
```

Comment:
This is wrong. Please add this to the `conflict` section. The `suggest` value should explain the suggested dependency.